src/packet/radiotap_header.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright nPrint 2021
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ */
+
+#include "radiotap_header.hpp"
+
+void *RadiotapHeader::get_raw() {
+    return (void *) raw;
+}
+
+void RadiotapHeader::set_raw(void *raw) {
+    this->raw = (struct radiotap_header *) raw;
+}
+
+void RadiotapHeader::print_header(FILE *out) {
+    if (raw == NULL) {
+        fprintf(out, "RadiotapHeader: raw data not set\n");
+    } else {
+        fprintf(out, "RadiotapHeader: %d bytes\n", get_header_len());
+    }
+}
+
+uint32_t RadiotapHeader::get_header_len() {
+    return 56;
```

Comment:
Remove the magic number and lets use a SIZE_ macro

Reply:
Agree, I just try to mimic the ethernet_header.cpp file.

https://github.com/nprint/nprint/blob/eb76192cb0d5dd606da479663d4782084c9f5148/src/packet/ethernet_header.cpp#L33

Reply:
Yep, this looks like my laziness :) let's fix both.

Reply:
I just noticed that RadioTap Header may have a variant length. I will figure a way to support it.